styleguide/markdown-syntax-style.md

```diff
@@ -1,157 +1,88 @@
-| Links                | Relative paths to other docs, full paths to anything else | Always full paths, check for HTTPS and that you’re not using .localhost | Always full paths, check for HTTPS and that you’re not using .localhost |
-| Exclamation marks    | No                                                        | Yes, although use with restraint, and not more than one in a row        | Yes, although use with restraint, and not more than one in a row        |
-
-
-### Glossary of notable terms and their spelling
-
-| Word                      | Usage                                                                                            |
-|---------------------------|--------------------------------------------------------------------------------------------------|
-| The Buildkite Agent/agent | When referring to the running process/piece of software as a whole                               |
-| `buildkite-agent`         | When referring to the CLI tool, visually should be presented in a code block                     |
-| Sign up/log in            | The action of signing up                                                                         |
-| Signup/login              | When referring to a page that enables signing up or to the signup process                        |
-| Time out/timeout          | Time out is a verb, timeout is a noun                                                            |
-| API, SSO, SAML            | Always capitalized                                                                               |
-| GitHub                    | Always capitalized, with an uppercase H in the middle                                            |
-| Two-factor authentication | In short form 2FA |
-| Single sign-on            | In short form SSO                       |
-
-### Common trip-ups
-
-Linters cannot do all of the work for you, so please pay attention to the following cases:
-
-* **Their/they’re/there, your/you’re** - [check](https://www.dictionary.com/e/their-there-theyre/ ) if you’ve got the right one for your situation!
-* **Affect/effect** - affect is a verb, effect is a noun. When you affect something, you’re impacting or changing the thing. When you have an effect, it’s the outcome or result of a change.
-* Be mindful of **hyphens**! Hyphens for compound adjectives, no hyphens in verbs: 'end-user documentation' vs. 'for the end user'.
-
-### Headings
-
-No multi-sentence headings or full stops at the end of a sentence in page or sentence headings.
-Be consistent about heading levels - H1, H2, H3 - no jumping from H1 to H3 or H2 to H4.
+| Links                | Relative paths to other docs, full paths to anything else | Always full paths, check for HTTPS and that you're not using .localhost | Always full paths, check for HTTPS and that you’re not using .localhost |
```

Comment:
It looks like the link information is also included in the Working with the site section

Reply:
Yeah, because I haven't been through the entire style guide content, there's some stuff I've not focussed on.

I'm thinking of having a separate section on 'Links' as part of the 'Style and formatting' section (both in the Writing and Markdown syntax style guides) on what text to link, and how to do it in Markdown, respectively.

Reply:
For the Docs column, spell out where contributors copy the URL slug from.

Reply:
Ditch table formatting for this and mention (in a note) when creating links to Twitter and Changelog from the Docs, do as instructed.

Reply:
I decided to completely refactor the existing 'Links' and 'Anchor links' sections of this page and move them higher up into the 'Style and formatting' section.